### ISSUE TITLE
Support ADS-defined astronomy journal macros

### ISSUE THREAD

**Issue by tomdonaldson** [ivoa-std/ivoatex#119]
When the reference below was added to [localrefs.bib](https://github.com/ivoa-std/VOTable/blob/657fbb1149a2a4ab3e0751f897b32e3b9cc9c284/localrefs.bib), it resulted in this error in building the document:
```
[31] (./VOTable.bbl [32]
! Undefined control sequence.
l.90 ...d relative dimension in space}', {\em \aap
                                                  } {\bf 574},~A36,
? )
```

New reference:
```
@ARTICLE{2015A&A...574A..36R,
       author = {{Rots}, Arnold H. and {Bunclark}, Peter S. and {Calabretta}, Mark R. and {Allen}, Steven L. and {Manchester}, Richard N. and {Thompson}, William T.},
        title = "{Representations of time coordinates in FITS. Time and relative dimension in space}",
      journal = {\aap},
     keywords = {time, standards, methods: data analysis, techniques: miscellaneous, astronomical databases: miscellaneous, reference systems, Astrophysics - Instrumentation and Methods for Astrophysics},
         year = 2015,
        month = feb,
       volume = {574},
          eid = {A36},
        pages = {A36},
          doi = {10.1051/0004-6361/201424653},
archivePrefix = {arXiv},
       eprint = {1409.7583},
 primaryClass = {astro-ph.IM},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2015A&A...574A..36R},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```

Thanks to @msdemlei for identifying the issue as ivoatex not yet knowing about some journal macros used by ADS (including `\aap`), and for the workaround for VOTable which is to insert the following line right after right after 
` \usepackage{verbatim}` in [VOTable.tex](https://github.com/ivoa-std/VOTable/blob/657fbb1149a2a4ab3e0751f897b32e3b9cc9c284/VOTable.tex#L4):
```
\newcommand{\aap}{Astronomy and Astrophysics}
```

**Comment by msdemlei** [comment]
On Mon, Sep 04, 2023 at 05:39:28AM -0700, Marco Molinaro wrote:
> I've seen this one http://cdsads.u-strasbg.fr/abs_doc/aas_macros.html
> Maybe ADS or CDS people have more sure answers.

Ah, thanks, that's what I was looking for.  (In particular
@tomdonaldson,) please review PR #121 and, while you're at it, the
related ivoatexDoc PR https://github.com/ivoa-std/ivoatexDoc/pull/35.

**Comment by msdemlei** [comment]
Fixed by PR #121
